middleware/kubernetes/kubernetes_test.go

```diff
@@ -212,36 +194,6 @@ func TestParseRequest(t *testing.T) {
 	for field, expected := range tcs {
 		expectString(t, f, "TXT", query, &r, field, expected)
 	}
-
-	// Invalid query tests
```

Comment:
I think if you do the tests here they show up in coverage, but not if you do them in the integration tests.